data/map planets.txt

```diff
@@ -404,6 +404,7 @@ planet Bluerun
 	attributes hai station uninhabited
 	landscape land/station33
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
+	description `	Bluerun seems to have been mainly built for civilian habituation, but extensive mineral processing facilities indicate that ore refining was once a sizable industry here. No doubt the designers of this station sought to take advantage of the large number of metallic asteroids in this system and process ores stationside rather than rely solely on Stonebreak's facilities. Doing so presumably reduced the resource burden caused by the massive influx of migrants to this station.`
```

Comment:
```suggestion
	description `	Bluerun seems to have been mainly built for civilian habitation, but extensive mineral processing facilities indicate that ore refining was once a sizable industry here. No doubt the designers of this station sought to take advantage of the large number of metallic asteroids in this system and process ores stationside rather than rely solely on Stonebreak's facilities. Doing so presumably reduced the resource burden caused by the massive influx of migrants to this station.`
```